### DISCUSSION TITLE
Strict transformations

### DISCUSSION THREAD

**Discussion by hazzyeer** [cloudinary-devs/gatsby-transformer-cloudinary#153] [Q&A]
Hello,
I would like to know if it's possible to enable strict transformations.
https://cloudinary.com/documentation/control_access_to_media#strict_transformations
Thank you.

**Comment by raae** [comment]
The transformations are generated with your graphql queries. I would imagine if you run your build once (generating all the transformations you need) you should be okey with turing strict transformations on.
